gustaf/helpers/data.py

```diff
@@ -297,7 +328,7 @@ class ComputedData(DataHolder):
     _depends = None
     _inv_depends = None
 
-    ___slots___ = []
+    __slots__ = ()
```

Comment:
I think sets must be initialized with set() and cannot be initialized with just round empty braces.

Reply:
just wanted a tuple instead of slots.